test/e2e/tests/01-required-and-optional.js

```diff
@@ -0,0 +1,261 @@
+}
+
+module.exports = {
+  'Begin application': (client) => {
+    console.log(url);
+
+    // Ensure introduction page renders.
+    client
+      .url(url)
+      .waitForElementVisible('body', timeouts.normal)
+      .assert.title('Apply for Health Care: Vets.gov')
+      .waitForElementVisible('.form-panel', timeouts.slow)  // First render of React may be slow.
+      .click('.form-panel .usa-button-primary');
+    expectNavigateAwayFrom(client, '/introduction');
+
+    // Personal Information page.
+    client.expect.element('input[name="fname"]').to.be.visible;
+    client
+      .setValue('input[name="fname"]', 'William')
+      .setValue('input[name="mname"]', 'Swan')
+      .setValue('input[name="lname"]', 'Shakespeare')
+      .setValue('select[name="suffix"]', 'Jr.')
+      .setValue('input[name="mothersMaidenName"]', 'Arden')
+      .click('.form-panel .usa-button-primary');
+    expectNavigateAwayFrom(client, '/veteran-information/personal-information');
+
+    // Birth information page.
+    client.expect.element('select[name="veteranBirthMonth"]').to.be.visible;
+    client
+      .setValue('select[name="veteranBirthMonth"]', 'Apr')
+      .setValue('select[name="veteranBirthDay"]', '23')
+      .setValue('input[name="veteranBirthYear"]', '1980')
+      .setValue('input[name="ssn"]', '111-22-3333')
+      .setValue('input[name="cityOfBirth"]', 'Akron')
+      .setValue('select[name="stateOfBirth"]', 'OH')
+      .click('.form-panel .usa-button-primary');
+    expectNavigateAwayFrom(client, '/veteran-information/birth-information');
+
+    // Demographic information page.
+    client.expect.element('select[name="gender"]').to.be.visible;
+    client
+      .setValue('select[name="gender"]', 'M')
+      .click('input[name="isAmericanIndianOrAlaskanNative"] + label')
+      .click('input[name="isBlackOrAfricanAmerican"] + label')
+      .click('input[name="isNativeHawaiianOrOtherPacificIslander"] + label')
+      .click('input[name="isAsian"] + label')
+      .click('input[name="isWhite"] + label')
+      .click('input[name="isSpanishHispanicLatino"] + label')
+      .click('.form-panel .usa-button-primary');
+    expectNavigateAwayFrom(client, '/veteran-information/demographic-information');
+
+    // Veteran Address page.
+    client.expect.element('input[name="address"]').to.be.visible;
+    client
+      .setValue('input[name="address"]', '111 S Michigan Ave')
+      .setValue('input[name="city"]', 'Chicago')
+      .setValue('select[name="country"]', 'USA')
+      .setValue('select[name="state"]', 'IL')
+      .setValue('input[name="zip"]', '60603')
+      .click('.form-panel .usa-button-primary');
+    expectNavigateAwayFrom(client, '/veteran-information/veteran-address');
+
+    // Contact Information Page.
+    client.expect.element('input[name="email"]').to.be.visible;
+    client
+      .setValue('input[name="email"]', 'bills@bard.com')
+      .setValue('.input-section > div:nth-child(2) input', 'bills@bard.com')
```

Comment:
I've updated three components with an `additionalClass` (rather than `-es`) prop: ErrorableTextInput and it's children that I care about for the moment: Email and Phone.